vassal-app/src/main/java/VASSAL/build/module/NotesWindow.java

```diff
@@ -45,21 +43,26 @@
 import VASSAL.configure.Configurer;
 import VASSAL.configure.ConfigurerFactory;
 import VASSAL.configure.IconConfigurer;
-import VASSAL.configure.NamedHotKeyConfigurer;
 import VASSAL.configure.TextConfigurer;
 import VASSAL.i18n.Resources;
 import VASSAL.tools.LaunchButton;
-import VASSAL.tools.NamedKeyStroke;
 
 /**
  * This is a {@link GameComponent} that allows players to type and
  * save text notes during a game.  There is one set of shared public
  * notes, and each player has a set of private notes visible only to
  * him
  */
-public class NotesWindow extends AbstractConfigurable
+public class NotesWindow extends AbstractToolbarItem
     implements GameComponent, CommandEncoder {
 
+  public static final String BUTTON_TEXT = "buttonText"; // non-standard legacy difference from AbstractToolbarItem
+
+  // These three identical to AbstractToolbarItem, and are only here for "clirr purposes"
+  public static final String HOT_KEY = "hotkey"; //$NON-NLS-1$
+  public static final String ICON = "icon"; //$NON-NLS-1$
+  public static final String TOOLTIP = "tooltip"; //$NON-NLS-1$
```

Comment:
Does it actually complain about these being moved to a superclass? (If so, could the ones here be marked deprecated?)

Reply:
Will mark them deprecated and hopefully nothing will get all deprecated-mad about that.